cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/BridgeConfiguration.java

```diff
@@ -24,11 +24,11 @@
     private Map<String, String> messages;
     private boolean logPlayerConnections = true;
     private boolean hubCommandEnabled = true;
-    private ArrayList<String> hubCommandAliases;
+    private Collection<String> hubCommandAliases;
```

Comment:
This may be final

Reply:
Anything else in this class isn't final either, I think a setter would be better. Without a setter, it would just be more difficult to modify the aliases.